os/include/arpa/inet.h

```diff
@@ -209,6 +224,9 @@ int         inet_aton(FAR const char *cp, FAR struct in_addr *inp);
 /**
  * @brief Convert the Internet host address from the IPv4 numbers-and-dots notation into binary data in network byte order.
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised